portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,18 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Becca's Portfolio</h1>
+        <p>Welcome!</p>
+        <p>I'm a rising junior computer science major at Vanderbilt University</p>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <p>Click here to get a random fun fact about me:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
+        <div class="image-thumbnails">
+            <p> random pic for now </p>
+        <a href="images/earth.jpg"><img src="images/earth.jpg"/></a>
```

Comment:
This is totally personal preference on my part, but any time you add a link that links outside of the site (e.g. if it's not navigating to another page on the site), I'd love if you set it with target=blank. I hate having to specifically tell things to open in new tabs lol.

https://www.w3schools.com/tags/att_a_target.asp

Reply:
done